dotCMS/src/main/java/com/dotcms/security/apps/AbstractProperty.java

```diff
@@ -43,4 +44,22 @@ public boolean getBoolean() {
         return Boolean.parseBoolean(getString());
     }
 
+    @Override
+    public boolean equals(Object o) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Parameter 'o' is not assigned and could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5289044)